samples/node/createAgent.js

```diff
@@ -0,0 +1,101 @@
+const fs = require('fs');
+const path = require('path');
+const credentials = require('./credentials');
+const SkySyncClient = require('skysync-cli').SkySyncClient;
+
+const client = new SkySyncClient(credentials);
+
+let nfsConnection, o365Connection, template, profile;
+
+async function addConnections() {
+	nfsConnection = await client.connections.add({
+		name: 'Sample NFS Connection',
+		platform: { id: 'nfs' },
+		disabled: true
```

Comment:
are the connections disabled because they only are needed for the template?

Reply:
exactly